bindings/generator/generate.py

```diff
@@ -158,6 +158,12 @@ def peek_next(index: int) -> str:
 env.filters["snake2camel"] = snake_to_camel_case
 env.filters["pascal2snake"] = pascal_to_snake_case
 
+for ty in ("struct", "enum", "variant"):
```

Comment:
Another point: I'd rather use regular function for complex transformation (here we are generation a string from a type structure) and use jinja filter only for simple obvious operations (e.g. indent, lowercase etc.)

Reply:
I provided this function as function in Jinja instead of filter and remove the loop that was generating those Jinja's filters.

If you prefer to have the code in `generate.py` mostly for data handling and keep the transformation stuff outside of it I could work something out.

Reply:
Leave the `struct_js_to_rs_function_name` & friend macros please

Reply:
I would have prefer that you where more direct with the review. 
Instead of sending mixed signal by _requesting some change_ where you clearly don't want that PR to be merged.
It would have been better to just close it with a message or leaving a comment.

Reply:
>  Instead of sending mixed signal by requesting some change where you clearly don't want that PR to be merged.

Please refrain from guessing people intention, I can assure you I don't work in bad faith ;-)

>  It would have been better to just close it with a message or leaving a comment.

From it name, I thought this PR was about replacing lowercase by snake case in the function name.
I told you in a previous PR that I you can do that change if you find this more readable.

Reviewing this PR I focused in the points I find showstopper (i.e. mixing code generation with data representation) and didn't realize the snake case conversion has already been implemented by another PR.

I realized now that this PR is only about those change, so indeed this PR is not needed. Next time name your PR correctly please.